### DISCUSSION TITLE
Ender 3v2 Skr mini e3 v3 frozen display

### DISCUSSION THREAD

**Discussion by ITenku** [mriscoc/Ender3V2S1#877] [General]
Hello,
I'm  currently trying  to switch to a  SKR mini e3 v3 on my Ender 3v2. 
Specs:
Ender 3 v2
Sprite Extrude Pro all Metal Hotend
CR Touch
SKR mini e3 v3
Rewired cable for the display
normal Ender 3 v2 display

I downloaded the version "Ender3V2-SKRME3V3-BLTUBL-MPC-20230312.bin",
installed it on the SKR mini, "firmware.bin" turned into "FIRMWARE.CUR", the display turned on and displayed the menu.
I can navigate the menu by turning the wheel on the display, but if I select any option by pressing the wheel, no sound is played, the screen freezes for 3 seconds, before displaying the software version and a link to the wiki. When that happens I also hear a faint clack in the CR touch, though it doesn't extend. Afterwards the display unfreezes and I can continue to select other options each with the same result.

**Comment by mriscoc** [comment]
Caution, your display wiring is causing a short circuit. This was answered before, please do a search.

![image](https://user-images.githubusercontent.com/2745567/214766100-0451882e-f731-4e70-9cb3-813ab7aebe07.png)
